channels.yaml

```diff
@@ -842,3 +851,9 @@ releases:
     serverArgs: *serverArgs-v9
     agentArgs: *agentArgs-v7
     featureVersions: *featureVersions-v1
+  - version: v1.31.8+k3s1
+    minChannelServerVersion: v2.9.0-alpha1
```

Comment:
To keep it consistent can we update this to 2.10 for v1.31 https://github.com/rancher/kontainer-driver-metadata/pull/1650/files#diff-cc2b16200b02fa28bd0b6e16e6d9b67c9f3974307f0ed7b8ed0cb27b0a1cd9f9R857

Reply:
Please ignore. We have been using v2.9.0-alpha1 as minChannelServerVersion for v1.31